package.json

```diff
@@ -87,11 +87,8 @@
         "packages/lwc",
         "playground"
     ],
-    "engines": {
-        "node": ">=10"
-    },
     "volta": {
-        "node": "20.12.2",
+        "node": "22.7.0",
```

Comment:
Locally, we may as well use a recent Node version. CI tests can still fail if we use something that doesn't work in Node 18.